src/lib/php-get-theme-details.ts

```diff
@@ -1,37 +1,42 @@
-import * as Sentry from '@sentry/electron/main';
+import { z } from 'zod';
 import SiteServerProcess from 'src/lib/site-server-process';
 
+export const themeDetailsSchema = z.object( {
+	name: z.string().catch( '' ),
```

Comment:
TIL!

Reply:
I included this because it wasn't 100% clear to me if `WP_Theme::get( 'Name' )` ([docs](https://developer.wordpress.org/reference/classes/wp_theme/get/)) is guaranteed to return a string. It seems theoretically possible that it would also return a boolean or array. Defaulting to an empty string in those cases seemed reasonable.